src/XMLSecEnc.php

```diff
@@ -1,11 +1,13 @@
 <?php
-namespace RobRichards\XMLSecLibs;
+
```

Comment:
There's no need to touch the old files, since they are going away. They are only kept here for now as a reference during reimplementation.